src/models/launch_lite_1/repairs.md

```diff
@@ -0,0 +1,158 @@
+
+### Steps to remove a switch:
+
+1. [Remove the keycap](#removing-and-installing-keycaps) on the switch you wish to remove.
+2. Place the two tongs of the switch removal tool into the switch's locking tabs on the top and bottom edges of the switch.
+
+![Switch removal](./img/switch-removal.webp)
+
+3. Pinch the tongs together to release the switch's locking clips from the chassis.
+4. Pull the switch upwards out of the socket.
+    - The switch's locking tabs can sometimes catch on the edges of the chassis cutouts. If the switch feels stuck, remove the tool and press the switch back down into place, then try removing it again.
+    - Switches take more force to remove than keycaps.
+
+![A removed switch](./img/switch-removed.webp)
+
+### Steps to install a switch:
+
+1. Make sure the switch's pins are straight.
+    - If either of the pins is bent, gently bend it back into place.
+    - **Caution:** Attempting to install a switch with bent pins may damage the PCB and/or switch.
+
+![Bent vs. straight switch pins](./img/switch-pins.webp)
+
+2. Orient the switch so the clear LED window aligns with the LED on the PCB, near the top of the socket.
+
+![Switch orientation](./img/switch-orientation.webp)
+
+3. Place the switch straight into the plate.
+    - **Caution:** Forcing the pins into the hot swap socket if the pins are not properly aligned could damage the PCB and/or switch.
+    - If you feel excessive resistance, pull the switch out, double-check that the pins are straight, and try installing the switch again.
+4. Once you feel the pins plug into the hot swap socket, press the switch down with more force to snap it into place within the chassis.
+
+![Switch installation](./img/switch-installation.webp)
+
+## Removing the PCB:
+
+**Tools required:** Keycap puller, MX-style switch puller, and cross-head (Phillips) screwdriver  
+**Time estimate:** 50 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+If the PCB in your Launch becomes damaged and needs to be replaced, or if you are changing all of the switches on the PCB, it can be removed and replaced using these instructions.
+
+### Steps to remove the PCB:
+
+1. [Remove all of the keycaps](#removing-and-installing-keycaps) and [all of the switches](#removing-and-installing-switches).
+    - The 2U stabilizers do not need to be removed.
+
+![Launch Lite with all keycaps removed](./img/all-keycaps-removed.webp)
+![Launch Lite with all switches removed](./img/all-switches-removed.webp)
+
+2. Turn the keyboard over and remove the four bottom panel screws.
+    - The screws are set into adapters that may or may not come out with the screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Remove the bottom panel.
+    - If the magnetic lift bar is installed, it can be used as a handle.
+    - A small plastic spacer (highlighted cyan below) may come loose; if it does, place it back in the middle PCB hole.
```

Comment:
The center plastic spacer (highlighted in cyan below) is not attached to the PCB. Remove and save to replace in center hole on installation of the PCB (new or old).

Reply:
It's supposed to snap into the hole, it's just somewhat easy to dislodge.

Reply:
@leviport For what it's worth, it did not "snap" at all on the unit I had for photos. I could tell from its shape that it was supposed to snap (after you explained it to me), but either the hole in the PCB was too large or the plastic piece was too small. It sounds like this may be a wider issue than just a lab unit quirk if Thomas also knew about it?

I do think the current description covers the case of the spacer not being attached and is simpler than the suggested update. The current description doesn't say anything about snapping. (The second sentence in the suggested update is missing a subject, too.)

Reply:
I'll consult Mechanical Engineering about it. There were some part substitutions being made for that spacer, and the keyboard I handed you was a pre-production unit, so the diameter of the spacer probably increased a bit to stay put a little better.

Reply:
Alright, I talked it over with John. I guess the plastic piece is a _spacer_ that doesn't really snap into the PCB very well. I think the wording here is probably good though.